zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -5044,6 +5102,44 @@ components:
           description: The version of the form.
           type: integer
           format: int64
+    AuthorizationSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      type: object
+      properties:
+        filter:
+          allOf:
+            - $ref: "#/components/schemas/AuthorizationFilterRequest"
+    AuthorizationFilterRequest:
+      type: object
+      properties:
+        ownerType:
```

Comment:
🔧 Let's add descriptions to properties in requests and responses so users can understand what they can use this for and what those properties mean.